Makefile

```diff
@@ -97,14 +97,16 @@ build-lambdas:
 	@echo -e "\n\n\n============================="
 	@echo "Install Node Modules in all packages"
 	@echo "============================="
-	npm install --production --prefix server
+	npm install --prefix server
 	npm install --production --prefix server/lambda/layer/common/nodejs/custom_modules
 	npm install --production --prefix server/lambda/layer/common/nodejs
 
 	@echo -e "\n\n\n============================="
 	@echo "Copy backend to temp folder to start build"
 	@echo "============================="
-	cp -r ./server ./terraform/build/server
+	npm run build --prefix server
+	rm -rf server/node_modules && npm install --production --prefix server
```

Comment:
we install at line 100, then why do we install again?

Reply:
well, we don't want to include devDependencies in the lambda zip. But we need to do a full install in order to run the build.